dispatcher_plugin_gw/queries.py

```diff
@@ -7,6 +7,14 @@
 from gwpy.timeseries.timeseries import TimeSeries
 
 from .products import SkymapProduct, SpectrogramProduct, StrainProduct
+from astropy.time import Time as astropyTime
+
+def check_time_interval(T1, T2, maxinterval=60):
+    t1 = astropyTime(T1, format='isot')
+    t2 = astropyTime(T2, format='isot')
+    delta = t2 - t1
+    if delta.sec > maxinterval:
+        raise ValueError(f'Too long time interval. Current limit is {maxinterval}s')
```

Comment:
can the `astropyTimeDelta` or the `TimeDelta` parameter be used here?

Reply:
delta here is actually astropy.time.TimeDelta